tyssue/dynamics/planar_vertex_model_env_pressure.py

```diff
@@ -0,0 +1,7 @@
+from . import effectors
```

Comment:
As for the specs, I think it is better to define the models at each simulation. It's not a lot of code, and avoids having too much code in the core library (I did not follow that advice always, but for the future...)